packages/experimental/Avatar/src/JSAvatar.tsx

```diff
@@ -4,6 +4,7 @@ import { JSAvatarProps, JSAvatarType, JSAvatarName, JSAvatarState } from './JSAv
 import { stylingSettings } from './JSAvatar.styling';
 import { compose, UseSlots, mergeProps, withSlots } from '@fluentui-react-native/framework';
 import { useAvatar } from './useAvatar';
+import { PresenceBadge } from '../../Badge';
```

Comment:
We should import this from the Badge package instead of using relative pathing, since the package can move in the future.

Reply:
Fixed. Thank for catching this